### ISSUE TITLE
e2e flake: backupnow failed: dial tcp ...: i/o timeout

### ISSUE THREAD

**Issue by hongchaodeng** [coreos/etcd-operator#349]
logs
```
time="2016-11-06T23:10:42Z" level=error msg="backupnow (test-etcd-16ss4-backup-tool:19999) request failed: Get http://test-etcd-16ss4-backup-tool:19999/backupnow: dial tcp 10.0.254.28:19999: i/o timeout" 
time="2016-11-06T23:10:42Z" level=error msg="check backup (test-etcd-16ss4-backup-tool:19999): unexpected status code (404 Not Found)" 
```

**Comment by xiang90** [comment]
@hongchaodeng Which e2e test is this from

**Comment by hongchaodeng** [comment]
Flakes in both disaster recovery tests where we create pvc

**Comment by hongchaodeng** [comment]
Some logs from last failure:
```
e2e_test.go:224: fail to make a latest backup: fail to request backupnow: backupnow (10.0.174.234:19999) request failed: Get http://10.0.174.234:19999/backupnow: dial tcp 10.0.174.234:19999: getsockopt: connection refused
```

It seems like connectivity issue.

**Comment by hongchaodeng** [comment]
@xiang90 

It's relying on service IP being propagated to jenkins slave machine. There could be a delay.

We can probably retry on such error.

**Comment by hongchaodeng** [comment]
fixed via #370
